src/main/java/uk/gov/companieshouse/authcodenotification/controller/AuthCodeNotificationController.java

```diff
@@ -16,13 +19,24 @@
 @RequestMapping("/company/{companyNumber}/auth-code")
 public class AuthCodeNotificationController {
 
+    @Autowired
+    private AuthCodeNotificationService authCodeNotificationService;
+
     @PostMapping("/send-email")
     public ResponseEntity<Object> sendEmail(@RequestHeader(value = ERIC_REQUEST_ID_KEY) String requestId,
                                             @PathVariable String companyNumber) {
-
         DataMap dataMap = new DataMap.Builder().companyNumber(companyNumber).build();
         ApiLogger.infoContext(requestId,"Request received for auth code email", dataMap.getLogMap());
 
+        try {
+            authCodeNotificationService.sendAuthCodeEmail(requestId, companyNumber);
+        } catch (ServiceException e) {
+            return new ResponseEntity<>(HttpStatus.INTERNAL_SERVER_ERROR);
+        } catch (Exception e) {
```

Comment:
Still not convinced we need this catch block for Exception (even though it's there in OE API). If it's removed, does code still compile, pass tests and run?

Reply:
Tests still pass.

Reply:
Suggests it can go. I'd just test with the call in Docker too though before removing.

Reply:
Removed and tested in docker